src/main/kotlin/com/sourcegraph/cody/autocomplete/action/AcceptAutocompleteActionHandler.kt

```diff
@@ -57,8 +57,14 @@ class AcceptAutocompleteActionHandler : AutocompleteActionHandler() {
     val project = editor.project ?: return
     val server = CodyAgent.getServer(project)
     if (server != null) {
-      val telemetry = CodyAutocompleteManager.instance.currentAutocompleteTelemetry
-      GraphQlLogger.logAutocompleteAcceptedEvent(project, telemetry?.params())
+      val telemetry = CodyAutocompleteManager.instance.currentAutocompleteTelemetry ?: return
+
+      val id = telemetry.logID
+      if (id != null) {
+        server.completionsAccepted(id)
+      }
+
+      // GraphQlLogger.logAutocompleteAcceptedEvent(project, telemetry?.params())
       server.autocompleteClearLastCandidate()
       acceptAgentAutocomplete(editor, maybeCaret)
     } else {
```

Comment:
@sourcegraph/jetbrains-plugin does anyone have any insight into why this `else` case might ever be hit? Im not sure when the agent server would ever be null if we're receiving completions

Reply:
The server may become null after receiving a completion, but I think it's an unlikely situation